src/DqtApi/ModelBinding/DateTimeReverseOrderBinder.cs

```diff
@@ -2,7 +2,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
 
-namespace DqtApi.ModelBinders
+namespace DqtApi.ModelBinding
```

Comment:
We do still need this Model Binder (for when we're data that is not within a JSON doc). There's not much that can easily be split out since this is binding `DateTime` and the other is binding `DateOnly` - I think the format would be the only thing.

Reply:
Moved the format string into `ModelBinding.Constants.DateFormat` and referenced it from both the model binder and the JSON converter.